lib/pbench/server/api/resources/upload_api.py

```diff
@@ -17,7 +17,7 @@
     States,
 )
 from pbench.server.filetree import DatasetNotFound, FileTree, Tarball
-from pbench.server.utils import IsoTimeHelper, filesize_bytes
+from pbench.server.utils import UtcTimeHelper, filesize_bytes
```

Comment:
Are we OK with the order of the imports on line 20?  🙃

Reply:
The bulk name change didn't stop to consider this, and for a case-sensitive sort it's arguably still correct. Are we going to define case-blind sorting for imports? That's always been ambiguous, and I wouldn't really be inclined to worry about that right now.

Reply:
Yeah, I was wondering the same thing...but perhaps we can kick this can further down the road.

Reply:
(Personally, I would prefer the case-blind approach.)